### PR TITLE
Added the files for Task 6. 

### PR DESCRIPTION
**Pull request by verginia**
Adjusted Readme file with requirements for task 6.

### CODE REVIEW

File: app/tasks/task6/form.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                <label for="email">Password:</label>
+                <input type="password" id="Password" name="password" placeholder="Enter your password (min. 8 characters)" minlength="8" required>
+            </div>
+            <div>
+                <label for="phone">Phone:</label>
+                <input type="tel" id="phone" name="phone" placeholder="Enter your phone number" required>
+            </div>
+            <div>
+                <label for="adress">Address:</label>
+                <input type="address" id="adress" name="adress" placeholder="Enter your adress" required>
```
Comment (ngoncearu) [inline app/tasks/task6/form.html:40]: input de tip adress nu exista. Cred ca ar trebui sa fie de tip text sau url daca merge vorba de o adresa web.

File: app/tasks/task6/form.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+<!-- Shift + Alt + F for formatting the code -->
+
+<body>
+    <h1>First form</h1>
+
+    <form action="#" method="post">
+        <fieldset>
+            <legend>Personal Info</legend>
+            <div>
+                <label for="name"> First Name:</label>
```
Comment (ngoncearu) [inline app/tasks/task6/form.html:19]: ``for="fname"`` ar trebui sa fie: valoarea id tre sa coincinda cu attributul for la label

File: app/tasks/task6/form.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                <input type="text" id="fname" name="name" placeholder="Enter your  first name" required>
+            </div>
+            <div>
+                <label for="name"> Last Name:</label>
```
Comment (ngoncearu) [inline app/tasks/task6/form.html:23]: ``for="lname"``

File: app/tasks/task6/form.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            </select>
+        </fieldset>
+
+        <fieldset>
+            <legend>Skills</legend>
+            <label>Skills:</label><br>
+            <label><input type="checkbox" name="skills" value="HTML"> HTML</label><br>
+            <label><input type="checkbox" name="skills" value="CSS"> CSS</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> JavaScript</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> Python</label><br>
```
Comment (ngoncearu) [inline app/tasks/task6/form.html:90]: ultimele 4 optiuni au acelasi value, hai sa ajustam

**Review by ngoncearu — CHANGES_REQUESTED** [review]

File: app/tasks/task6/form.html (hunk 1 above)
Comment (verginia) [reply app/tasks/task6/form.html:40]: fixed, changed to type="text"

File: app/tasks/task6/form.html (hunk 2 above)
Comment (verginia) [reply app/tasks/task6/form.html:19]: fixed, for="fname", id="fname" and name="fname", the same was fixed for "lname"

File: app/tasks/task6/form.html (hunk 3 above)
Comment (verginia) [reply app/tasks/task6/form.html:23]: fixed, for="lname", id="lname" and name="lname"

File: app/tasks/task6/form.html [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+            <div>
+                <label for="name"> Last Name:</label>
+                <input type="text" id="lname" name="name" placeholder="Enter your last name" required>
+            </div>
+            <div>
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" placeholder="Enter your email" required>
+            </div>
+            <div>
+                <label for="email">Password:</label>
```
[the remark this answers is not in the document]
Comment (verginia) [reply app/tasks/task6/form.html:31]: fixed, for="password", id="password" and name="password"

File: app/tasks/task6/form.html (hunk 4 above)
Comment (verginia) [reply app/tasks/task6/form.html:90]: adjusted

File: app/tasks/task6/form.html [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+            <label><input type="checkbox" name="skills" value="HTML"> HTML</label><br>
+            <label><input type="checkbox" name="skills" value="CSS"> CSS</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> JavaScript</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> Python</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> UNIX</label><br>
+            <label><input type="checkbox" name="skills" value="JavaScript"> VIM</label><br>
+        </fieldset>
+
+        <fieldset>
+            <legend>Describe any additional skills you have:</legend>
```
[the remark this answers is not in the document]
Comment (verginia) [reply app/tasks/task6/form.html:96]: done

**Comment by ngoncearu** [comment]
let's fix conflicts and merge it! good job!

**Pull request merged**